src/main/java/org/sonarlint/intellij/analysis/DefaultSonarQubeRunnerFacade.java

```diff
@@ -99,6 +99,7 @@ private void tryStart(String serverUrl, boolean preferCache) {
     globalProps.setProperty(SonarLintConstants.SONAR_URL, serverUrl);
     globalProps.setProperty(SonarLintConstants.ANALYSIS_MODE, SonarLintConstants.ANALYSIS_MODE_ISSUES);
     globalProps.setProperty(SonarLintConstants.VERBOSE_PROPERTY, Boolean.toString(projectSettings.isVerboseEnabled()));
+    globalProps.setProperty(SonarLintConstants.USE_WS_CACHE, Boolean.toString(true));
```

Comment:
I'm passing this to the global container as well, just in case we want to have it configurable as well, in the future